README.md

```diff
@@ -105,6 +105,69 @@ The navigation system comprised SLAM provided by the Cartographer online node an
 The global plan was generated by the default global planner known simply as global_planner.
 </p>
 
+### Control
+
+The robot needs to be able to navigate unknown environments, a robust and reliable control system is needed to achieve this.
+
+![](https://github.com/D7039E-E7032E/Project-Report/blob/Controll-delen-till-rapport/images/Control_Str.svg)
+
+In the illustration above the structure of the control system an its supporting components. 
+
+#### Global Navigator
+
+The purpose of the global navigator is to find a path the robot should follow to reach its goal. The the goal points is set manually and when the robot reaches it, a new one will be chosen by the operator. It will use information from the global map, a map of the combined scans from all robots. This is implemented using the already existing ROS package move_base that calculates a global path for the robot using a form of A-star algorithm. 
+
+The Global Navigator uses maps generated from cartographer, which is a slow process, combined with A-star not being a particular fast algorithm. This makes the Global Navigator a rather slow. This is solved with a Local Navigator.
```

Comment:
Change this section. We are making use of Dijkstras algorithm to create a global path plan. This is more resource intense than using a faster A* algorithm.